src/track/keyutils.cpp

```diff
@@ -16,8 +16,9 @@ static const char* s_openKeyPattern = "(1[0-2]|[1-9])([dm])";
 // Lancelot notation, the numbers 1-12 followed by a (minor) or b (major).
 static const char* s_lancelotKeyPattern = "(1[0-2]|[1-9])([ab])";
 
-// a-g followed by any number of sharps or flats.
-static const char* s_keyPattern = "([a-g])([#♯b♭]*m?)";
+// a-g followed by any number of sharps or flats, optionally followed by
+// a scale spec (m = minor, min, maj)
+static const char* s_keyPattern = "([a-g])([#♯b♭]*)(m|min|maj)?";
```

Comment:
Ah, sorry, is it already matched, because it starts with "min"?  

Please add a test case for "major" and "minor" 

I have just tested the expression at https://regex101.com/
and "gmaj" it matches for just "m" which is minor.  

"([a-g])([#♯b♭]*)(min|maj|m)?" 
works.